python-package/geobr/lookup_muni.py

```diff
@@ -0,0 +1,71 @@
+""" Lookup municipality codes and names
```

Comment:
Hey,

Can you follow the documentation convention from the other functions? 

Actually the function description has to be below the function statement. Take a look on the other functions :)